src/components/EventCreator.tsx

```diff
@@ -36,21 +36,43 @@ export function EventCreator(props: { onSave: () => Promise<void> }) {
           outcome: "",
         },
       ],
-      // event_maturity: new Date(Date.now() + 864e5).toISOString().split("T")[0], // default to a day from now
+      event_maturity: undefined,
     },
   });
 
   const handleSubmit: SubmitHandler<EventForm> = async (f: EventForm) => {
+    setError(undefined);
     try {
       const outcomes = f.outcomes.map((o) => o.outcome);
-      const event_maturity = new Date(Date.now()).getTime() / 1000;
+      // Make sure outcomes are unique
+      if (new Set(outcomes).size !== outcomes.length) {
+        throw new Error("Outcomes must be unique");
+      }
+
+      let event_maturity = f.event_maturity;
+
+      // If there's no event_maturity set, default to one minute from now
+      if (!event_maturity) {
+        console.log("no event maturity");
+        event_maturity = new Date(Date.now() + 60 * 1000).toISOString();
+      }
```

Comment:
think we should just error if none set

Reply:
if you pick a date it's gonna be set to midnight, so you have to wait a whole day to do an observe

Reply:
if we do a minute from now then it'll immediately expire and they won't be able to do anything, so we should at least pick a better default

Reply:
oh good point okay I'll go back to required